codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -133,8 +140,15 @@ public int initialWindowSize(Http2Stream stream) {
         return state(stream).initialWindowSize();
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * This method must be called from the same event loop context as the {@link ChannelHandlerContext}
```

Comment:
@nmittler - The thought was that you could easily have a wrapper or extension which provides the thread safe capability for the user (check if on eventloop, if not execute a runnable). So the interface is not necessarily limited...but instead just this implementation.